src/components/HomeMain.component/Categories.component/HomeCategories.jsx

```diff
@@ -0,0 +1,16 @@
+import racket from "../../../Assets/images/racquet.png";
+import React from "react";
```

Comment:
As a suggestion you don't need to import react explicitly.

Reply:
sorry ! that was a part of code snippet. I'll Remove that